greenwave_monitor/greenwave_monitor/ncurses_frontend.py

```diff
@@ -59,7 +60,7 @@ def __init__(self):
         self.input_buffer = ''
         self.status_message = ''
         self.status_timeout = 0
-        self.show_only_monitored = False
+        self.show_only_monitored = hide_unmonitored
```

Comment:
Nit but we should pick one name or the other, I don't care too much which (but I'll vote for show_only_monitored)